packages/cms-lib/lib/errorHandlers/customErrors.js

```diff
@@ -0,0 +1,14 @@
+const getErrorMessage = error => {
+  const { category } = error;
+
+  switch (category) {
+    case 'MISSING_SCOPES':
+      return 'Your personal CMS access key is missing required permissions for this action. Your account may not have these permissions or you may need to generate a new access key to add them. You can do so by running "hs auth personalaccesskey". If you believe this is in error, please contact your administrator.';
```

Comment:
> Your personal CMS access key is missing required permissions for this action.
This sentence assumes that an access key is being used for authentication.

> Your account may not have these permissions or you may need to generate a new access key to add them. You can do so by running "hs auth personalaccesskey".

This is a situation where "account" is unclear. Are we referring to the hub or the user? If it is the hub, the action may be to either purchase CMS Hub enterprise, create a trial, or leverage a developer sandbox. I wonder if we should add some sort of backend API that we can use to determine if the issue is a portal issue or a user permission issue.

Reply:
@gchomatas do you think it would be possible to return more info along with the error specifying whether the `MISSING_SCOPES` are a user or hub problem?